src/_includes/footer.liquid

```diff
@@ -0,0 +1,66 @@
+<footer class="footer">
+    <div class="content">
+        <div class="columns">
+            <div class="column is-half is-offset-one-quarter">
+                <div class="field is-grouped is-grouped-centered">
+                    <p class="control">
+                        <a title="Discord" class="button is-discord is-rounded" target="_blank" href="{{ site.discord_community }}">
+                            <span class="icon"><i class="fab fa-discord"></i></span>
+                        </a>
+                    </p>
+
+                    <p class="control">
+                        <a title="GitHub" class="button is-github is-rounded"
+                           target="_blank"
+                           href="https://github.com/{{ site.github_username }}">
+                            <span class="icon"><i class="fab fa-github"></i></span>
+                        </a>
+                    </p>
+
+                    <p class="control">
+                        <a title="Twitter" class="button is-twitter is-rounded"
+                           target="_blank"
+                           href="https://twitter.com/{{ site.twitter_username }}">
+                            <span class="icon"><i class="fab fa-twitter"></i></span>
+                        </a>
+                    </p>
+
+                    <p class="control">
+                        <a title="Twitch" class="button is-primary is-rounded" target="_blank"
+                           href="https://twitch.tv/{{ site.twitch_username }}">
+                            <span class="icon"><i class="fab fa-twitch"></i></span>
+                        </a>
+                    </p>
+                </div>
+
+                <br />
+
+                <div class="has-text-centered">
+                    <p>
+                        <strong>Copyright &copy; {{ 'now' | date: "%Y" }} Quilt Project</strong>
+                    </p>
+
+                    <p>
+                        Quilt is not an official Minecraft product, and is not
+                        approved by or associated with Mojang Studios.
+                        "Minecraft" is a trademark of Mojang AB and any usage
```

Comment:
what is the difference between Mojang Studios and Mojang AB?

Reply:
Mojang AB is the legal entity, Mojang Studios is a division within Microsoft